frontend/test/specs/utils/hideNavbarSpec.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+(describe("Test hiveNavbar", function() {
+    beforeEach(module('app'));
+
+    let scope, hideDirective, STATES_CONST, elem, state;
+    beforeEach(inject(function($rootScope, $compile, STATES, $state) {
+        STATES_CONST = STATES;
+        state = $state;
+        scope = $rootScope.$new();
+
+        elem = angular.element('<div hide-navbar="bottom"></div>');
+        elem = $compile(elem)(scope);
+        hideDirective = elem.scope();
+        $rootScope.$digest();
+    }));
+
+    describe('isTopToolbarAllowed', function(){
+        it("should be true", function() {
+            state.current.name = STATES_CONST.NOTIFICATION;
+            expect(hideDirective.isTopToolbarAllowed()).toBeTruthy();
+        })
+        it("should be false", function() {
```

Comment:
you should verify for all states that should be false. you can do a test with "for" loop